api/lib/infrastructure/repositories/certification-repository.js

```diff
@@ -42,17 +42,37 @@ const publishCertificationCoursesBySessionId = async function (sessionId) {
     .onConflict('id')
     .merge(['pixCertificationStatus', 'isPublished', 'updatedAt']);
 };
+const getStatusesBySessionId = async function (sessionId) {
```

Comment:
Ajout d'un petit espace au dessus de la lambda :) 
```suggestion
};

const getStatusesBySessionId = async function (sessionId) {
```

Reply:
Done